packages/message-bus/package.json

```diff
@@ -10,14 +10,21 @@
     "url": "git+https://github.com/ConsenSys/web3studio-sidejam.git"
   },
   "scripts": {
-    "build": "yarn buidl",
-    "buidl": "truffle compile",
+    "test": "sidejam-truffle-test",
+    "build": "truffle compile",
```

Comment:
No more buidling anymore?

![bitmoji](https://render.bitstrips.com/v2/cpanel/5573e20d-d6a8-4d1f-b640-afc9250bfb18-a8384b5a-d582-44c0-9036-5456ef16dfbd-v1.png?transparent=1&palette=1&width=246)

Reply:
I guess I missed why 'buidling' is a thing in the ethereum community. Shall we keep it in?